types_test.go

```diff
@@ -47,22 +47,6 @@ var (
 				return int32(rand.Int())
 			},
 		},
-		{
```

Comment:
@fzerorubigd I removed the uint tests because it seemed most consistent with the general api. I discovered it doesn't support encoding other uint types like uint8/uint16. It seemed more consistent to always require the encoder parquet type than to allow all possible types of inputs.